R/utils.R

```diff
@@ -2,6 +2,17 @@ last <- function(x) x[length(x)]
 
 sq <- function(x) glue::single_quote(x)
 
+sglue <- function(...) stop(glue::glue(..., .envir = parent.frame()), call. = FALSE)
```

Comment:
For some reason I get weird behavior if I don't include the `.envir = parent.frame()`. If you run this does it work?
```
mglue <- function(...) message(glue::glue(...))
x <- function(val) mglue(val)
x("test")
```
On my machine it can't find `val` unless I explicitly set the `.envir`.